variables.tf

```diff
@@ -176,11 +180,23 @@ variable "node_pools" {
 }
 
 variable "node_pool_overrides" {
+  description = "Param override for var.node_pools"
+  type = map(
+    object({
+      compute  = map(string)
+      gpu      = map(string)
+      platform = map(string)
+    })
+  )
   default = {}
 }
 
 variable "namespaces" {
-  type        = object({ platform = string, compute = string })
+  type = object(
```

Comment:
what caused the formatting change? Are you using tf 1.x?

Reply:
Updated the format, `tf_fmt` is happy with it.